dbms/tests/queries/0_stateless/00502_sum_map.sql

```diff
@@ -7,6 +7,7 @@ INSERT INTO sum_map VALUES ('2000-01-01', '2000-01-01 00:00:00', [1, 2, 3], [10,
 
 SELECT * FROM sum_map ORDER BY timeslot;
 SELECT sumMap(statusMap.status, statusMap.requests) FROM sum_map;
+SELECT sumMap((statusMap.status, statusMap.requests)) FROM sum_map;
```

Comment:
Documentation for sumMap has to be updated.

Reply:
Fixed the documentation for sumMap in english version